clients/drcachesim/tools/filter/record_filter.cpp

```diff
@@ -143,6 +137,20 @@ record_filter_tool_create(const std::string &output_dir, uint64_t stop_timestamp
                 new dynamorio::drmemtrace::trim_filter_t(trim_before_timestamp,
                                                          trim_after_timestamp)));
     }
+    if (encodings2regdeps) {
+        /* If we are changing the encoding of trace_entry_t with encodings2regdeps, we
+         * will be generating discrepancies between encoding size and instruction length.
+         * So, we need to tell reader_t, which here comes in the form of memref_counter_t,
+         * to ignore such discrepancies. We do so by adding OFFLINE_FILE_TYPE_ARCH_REGDEPS
+         * to the file type of the filtered trace. Note that simulators that deal with
+         * these filtered traces will also have to handle the fact that
+         * encoding_size != instruction_length.
```

Comment:
This comment belongs in the filter, not here, right?

Reply:
Yes, makes more sense to have it in encodings2regdeps.h. Moved there. Thanks!